auth/api/iam/jar.go

```diff
@@ -123,7 +123,8 @@ func (j jar) Parse(ctx context.Context, ownDID did.DID, q url.Values) (oauthPara
 				return nil, oauth.OAuth2Error{Code: oauth.InvalidRequestURI, Description: "failed to get Request Object", InternalError: err}
 			}
 		case "post":
-			md, err := authorizationServerMetadata(ownDID)
+			issuerURL := ownDID.URI().URL
```

Comment:
which scheme does the URL get?

Reply:
`did`...

It's used for the wallet's client metadata, which in case of a mobile phone doesn't really have a (reachable) HTTPS endpoint... I'd think this would be actually SIOP-ish (`https://self-issued.me` or something).